src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -176,6 +178,10 @@ Result Device::ConfigureExtensions(const grfx::DeviceCreateInfo* pCreateInfo)
     }
 #endif
 
+    if (ElementExists(std::string(VK_KHR_PUSH_DESCRIPTOR_EXTENSION_NAME), mFoundExtensions)) {
```

Comment:
Do we currently have a mechanism for applications to specify which features they want and only request the necessary extensions?

Reply:
There isn't. It was left out when BigWheels was first written because we were basing everything off the primary platform's feature sit. Which was fixed. It may make sense now to add in a Vulkan extension section into the `ApplicationSettings` and have ti pass off onto the instance and device create infos.

Reply:
got it, filed https://github.com/google/bigwheels/issues/238 to keep track